tests/Integrations/Symfony/V2_3/RouteNameTest.php

```diff
@@ -13,6 +13,13 @@ protected static function getAppIndexScript()
         return __DIR__ . '/../../../Frameworks/Symfony/Version_2_3/web/app.php';
     }
 
+    protected static function getEnvs()
```

Comment:
Since this is used for several of the tests, we could possibly extract this to a trait if you think that would be better. :)

Reply:
Good idea. BTW given that these tests all are web framework tests, what if I can move directly there instead of a separate trait? In the end ALL web framework tests have to be tested with this env, WDYT?